src/DynamoCoreWpf/UI/Prompts/DynamoMessageBox.xaml.cs

```diff
@@ -168,56 +201,51 @@ public static MessageBoxResult Show(string messageBoxText, string caption)
         /// Shows or hides buttons on the view as necessary.
         /// </summary>
         /// <param name="messageBoxButton"></param>
-        private void ConfigureButtons(MessageBoxButton messageBoxButton)
+        /// /// <param name="buttonNames">names that will be used to override the standard button names.
+        /// Number of names must match the number of visible buttons set by messageBoxButton parameter.</param>
+        internal void ConfigureButtons(MessageBoxButton messageBoxButton, IEnumerable<string> buttonNames = null)
         {
             switch (messageBoxButton)
             {
                 case MessageBoxButton.OK:
+                    if(buttonNames!=null && buttonNames.Count() == 1)
+                    {
+                        OkButton.Content = buttonNames.ElementAt(0);
+                    }
                     OkButton.Visibility = Visibility.Visible;
                     break;
                 case MessageBoxButton.OKCancel:
+                    if (buttonNames != null && buttonNames.Count() == 2)
+                    {
+                        OkButton.Content = buttonNames.ElementAt(0);
+                        CancelButton.Content = buttonNames.ElementAt(1);
+                    }
                     OkButton.Visibility = Visibility.Visible;
                     CancelButton.Visibility = Visibility.Visible;
                     break;
                 case MessageBoxButton.YesNoCancel:
+                    if (buttonNames != null && buttonNames.Count() == 3)
+                    {
+                        YesButton.Content = buttonNames.ElementAt(0);
+                        NoButton.Content = buttonNames.ElementAt(1);
+                        CancelButton.Content = buttonNames.ElementAt(2);
+                    }
```

Comment:
same question here

Reply:
checkout the xaml for `DynamoMessageBox` - each button has standard content like
` Content="{x:Static p:Resources.GenericTaskDialogOptionCancel}"`